integreat_cms/cms/views/utils/search_content_ajax.py

```diff
@@ -44,9 +45,19 @@ def format_object_translation(
     :param typ: The type of this object
     :return: A dictionary with the title, path, url and type of the translation object
     """
+    if isinstance(object_translation.link_title, str):
```

Comment:
What is the difference between `object_translation.link_title` and `object_translation.title`?

Reply:
`link_title` is the method I defined in `integreat_cms/cms/models/abstract_content_translation.py` which can both return a `str` and an `Element`. I did this because the issue for this pr wants icons displayed next to the links for internal content translations. So in those cases, `link_title` returns the icon as an HTML Element in addition to the normal title